src/ui/index.ts

```diff
@@ -66,12 +66,18 @@ fin.desktop.main(async () => {
             const plugin = await pluginP;
             const payload = {uuid};
             const appNotifications = await plugin.dispatch('fetch-app-notifications', payload);
+            appNotifications.forEach(n => {
+                n.date = new Date(n.date);
+            });
             console.log('appNotifications', appNotifications);
         },
-        fetchAllNotifications: async () => {
+        fetchAllNotifications: async() => {
```

Comment:
Lint?

Reply:
looks like UI isn't being linted, making a note, will do in another PR.

Reply:
added src/ui/index.ts to tsconfig.json